manage

```diff
@@ -599,27 +690,17 @@ Or on Discord: \e[94m\e[4mhttps://discord.gg/RXSTvaUvuu\e[0m" | tee -a "$logfile
   #exit the manage script with the same exit-code of the app's script
   exit $exitcode
   
-elif [ "$1" == 'update' ];then #The manage script no longer handles updates. This mode is only for backwards-compatibility and uses the updater script
-  
-  warning "The manage script ONLY updates apps, and this mode has been replaced by the updater script.
-If you want to update Pi-Apps from the command-line, please use:
-~/pi-apps/updater cli-yes"
-  
+elif [ "$1" == 'update' ] || [ "$1" == 'refresh' ];then
```

Comment:
`update_file` has no corresponding action in the manage script. nothing will run for the files

Reply:
Manage daemon will avoid running manage at all for update_file.

Reply:
See line 479.

Reply:
yeah I saw that. that only occurs if you run the action through the manage daemon. if you simply wanted to use manage to update files you can't do that with your changes.